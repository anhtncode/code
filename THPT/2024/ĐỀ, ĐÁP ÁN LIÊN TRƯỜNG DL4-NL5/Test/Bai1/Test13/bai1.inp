1000000000000000 662654012256859 43318580548340