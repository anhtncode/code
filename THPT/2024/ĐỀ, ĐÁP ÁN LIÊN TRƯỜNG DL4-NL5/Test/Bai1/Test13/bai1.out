619335431708519 662654012256859